package.json

```diff
@@ -20,8 +20,8 @@
     },
     "scripts": {
         "build": "rollup -c",
-        "test": "node --experimental-modules node_modules/c8/bin/c8 --reporter=html --reporter=lcov --reporter=text --check-coverage node --experimental-modules node_modules/mocha/bin/mocha",
-        "coverage": "c8 report --reporter=text-lcov | coveralls",
+        "test": "c8 --report=none --no-clean ava",
+        "coverage": "c8 report | coveralls",
```

Comment:
It will not run on Node 10.x and 12.x without flags

Reply:
That's what I thought :confounded: I just copied the configuration from [AVA](https://github.com/avajs/ava).

Reply:
@tinovyatkin I updated the configuration to match the old, working behavior.